src/modules/Modal/Modal.js

```diff
@@ -157,7 +157,16 @@ class Modal extends Component {
     )
     const rest = getUnhandledProps(Modal, this.props)
     const ElementType = getElementType(Modal, this.props)
-    const portalProps = _.pick(rest, _.keys(Portal.propTypes))
+
+    // Remove portal-handled props from 'rest' and pass them through to Portal
+    const portalProps = _.reduce(rest, (memo, value, key) => {
```

Comment:
Just a suggestion here.  This adds another loop, but is a little clearer (at least to my eyes) and does not mutate any objects:

```jsx
const unhandled = getUnhandledProps(Modal, this.props)

const portalProps = _.pick(unhandled, _.keys(Portal.propTypes))
const rest = _.omit(unhandled, _.keys(portalProps))
```